tracer/test/Datadog.Trace.TestHelpers/EnvironmentHelper.cs

```diff
@@ -206,6 +206,8 @@ public static void ClearProfilerEnvironmentVariables()
                 environmentVariables["COR_PROFILER_PATH"] = nativeLoaderPath;
             }
 
+            environmentVariables["DD_WRITE_INSTRUMENTATION_TO_DISK"] = profilerEnabled;
```

Comment:
I see both DD_WRITE_INSTRUMETNATION_TO_DISK in code but DD_**TRACE**_WRITE_INSTRUMENTATION_TO_DISK in the docs?

Reply:
Fixed in 59dc3c9f22d7700ec54e60483dd11a150535c39c